parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueReaders.java

```diff
@@ -137,6 +137,41 @@ public void setPageSource(PageReadStore pageStore) {
     }
   }
 
+  static class PositionReader implements ParquetValueReader<Long> {
+    private long rowOffsetInCurrentRowGroup = -1;
+    private long rowGroupRowOffsetInFile;
```

Comment:
In general, try to be specific with names, but avoid unnecessary context. In this case, these names can be simpler: `rowGroupStart` and `rowOffset` would work fine. Extra context like `InFile` and `InCurrent` aren't adding clarity.

Reply:
Done.